src/types.ts

```diff
@@ -128,7 +129,7 @@ export class Row implements MetaDataBaseObject {
       attemptId : number
     },
     state : string,
-    startTstmp : string,
+    startTStamp : string,
```

Comment:
Timestamp is one word, so it doesn't need camel case and could be left as startTstamp ;-)

Reply:
Actually that has been changed because of a typo I made in the script creating random databases... All fake statefiles had a "startTStamp", but the correct format (according to SDLB statefiles) is indeed "startTstmp".